README.md

```diff
@@ -461,9 +461,12 @@ The `addJob` arguments are as follows:
   - `queueName`: the queue to run this task under
   - `runAt`: a Date to schedule this task to run in the future
   - `maxAttempts`: how many retries should this task get? (Default: 25)
-  - `jobKey`: unique identifier for the job, used to update or remove it later
-    if needed (see [Updating and removing jobs](#updating-and-removing-jobs));
-    can also be used for de-duplication
+  - `jobKey`: unique identifier for the job, used to replace, update or remove
+    it later if needed (see
+    [Updating and removing jobs](#updating-and-removing-jobs)); can be used for
+    de-duplication (i.e. throttling or debouncing)
+  - `jobKeyMode`: controls the behavior of `jobKey` when a matching job is found
+    (see [Updating and removing jobs](#updating-and-removing-jobs))
```

Comment:
Internal heading #link won't work, because of renamed heading (same issue above).
```suggestion
    (see [Replacing, updating and removing jobs](#replacing-updating-and-removing-jobs))
```